tests/bindings/python/test_units.py

```diff
@@ -0,0 +1,71 @@
+#
+# Tests the Units class bindings
+#
+import sys
+import unittest
+
+class UnitsTestCase(unittest.TestCase):
+
+    def test_model(self):
+        from libcellml.units import Units
+        
+        # Test create/copy/destroy
+        x = Units()
+        del(x)
+        y = Units()
+        z = Units(y)
+        del(y,z)
+        
+        # Test inheritance
```

Comment:
Ditto here, I would create a new function, e.g. `test_model_inheritance` or the like.